installer/conf/sudoers

```diff
@@ -29,4 +29,8 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
 
 #run hdinsightmanifestreader.rb
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsoft/omsagent/bin/hdinsightmanifestreader.rb
+
+#run update management commands
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * 
```

Comment:
@shuliums @robbiezhang 
If we are not doing any remove/purge, and only doing install, lets only have the require params explicitly -
```
/usr/bin/apt-get install *
/usr/bin/apt-get update *
/usr/bin/apt-get upgrade *
/usr/bin/apt-get dist-upgrade *
```